### ISSUE TITLE
Remove site key property from reference representation

### ISSUE THREAD

**Issue by jimmyca15** [Microsoft/IIS.Administration#113]
The reference representation of an object is the abbreviated version that appears when the resource is embedded into another resource. Only select fields are displayed to make the resource human recognizable. The site key was added to the website reference representation and it should be removed to prevent bloating of this abbreviated view.

in 2.0.0 the website reference representation appears as thus
```
"website": {
    "name": "Default Web Site",
    "id": "o06wRhVwLlRQKMtJ876dLQ",
    "key": "1",
    "status": "started",
    "_links": {
        "self": {
            "href": "/api/webserver/websites/o06wRhVwLlRQKMtJ876dLQ"
        }
    }
}
```

it **should** be
```
"website": {
    "name": "Default Web Site",
    "id": "o06wRhVwLlRQKMtJ876dLQ",
    "status": "started",
    "_links": {
        "self": {
            "href": "/api/webserver/websites/o06wRhVwLlRQKMtJ876dLQ"
        }
    }
}
```

**Comment by jimmyca15** [comment]
The site key can still be retreived when using the web sites API through the use of the _fields_ query string paramater

ex: GET _/api/webserver/websites?fields=name,status,key_

**Comment by jimmyca15** [comment]
Fixed in #121
